zetaclient/orchestrator/bootstrap.go

```diff
@@ -481,6 +479,20 @@ func syncObserverMap(
 	return added, removed, nil
 }
 
+func btcDatabaseFileName(chain chains.Chain) string {
+	// legacyBTCDatabaseFilename is the Bitcoin database file name now used in mainnet and testnet3
+	// so we keep using it here for backward compatibility
+	const legacyBTCDatabaseFilename = "btc_chain_client"
+
+	// For additional bitcoin networks, we use the chain name as the database file name
+	switch chain.ChainId {
+	case chains.BitcoinMainnet.ChainId, chains.BitcoinTestnet.ChainId:
+		return legacyBTCDatabaseFilename
+	default:
+		return fmt.Sprintf("%s_%d", legacyBTCDatabaseFilename, chain.ChainId)
```

Comment:
Should updated accordingly (to chain name)

Reply:
updated: https://github.com/zeta-chain/node/pull/3224/commits/f10949c1e3dd31f1fc2943657e42b613fd4e2823